dts/bindings/radio_fem/skyworks,sky66112-11.yaml

```diff
@@ -61,3 +61,8 @@ properties:
             can be changed if, instead of SKY66112-11, another device with
             different gain, but compatible with the control interface via the
             CTX, CRX pins is used.
+    ant-sel-gpios:
```

Comment:
Nit: you could use an empty line before this and also use `|` instead of `>` to span the description, just for consistency with other properties defined here.

Reply:
Thanks, fixed.